styles/main/_justified_layout.scss

```diff
@@ -13,6 +13,7 @@
 
 .justified-layout > .photo {
 	position: absolute;
+	height: 320px;
```

Comment:
If our default `small_max_height` is 360, should this not be 360 too? Am I missing something?

Reply:
Well, the default of the justified-layout package that we use is 320px, and that's what we've been using in Lychee all along. I didn't want to change it since it would affect the look of people's existing albums.

As far as I can tell, the default value of `small_max_height` is arbitrary. It does not influence the layout, it simply determines the rendering quality. Perhaps it was set to 360 by mistake, or somebody decided that that would be "good enough" (you may know better than I do; that value predates my involvement in the project).

I agree that setting it to 320 would be less arbitrary but it would not necessarily be optimal. To keep the right margin justified, justified-layout can stretch an image by up to 2x, so to ensure the optimal rendering quality we should set it to 640. In a way, we already generate those higher-resolution variants and store in "@2x", and even on low-DPI displays the browser will currently use them for images stretched beyond 360px. But on HiDPI displays we do need to set `small_max_height` higher than 320. In principle, no value is guaranteed to be optimal as displays can be arbitrarily dense of course; we currently make provisions for double density and that's it.

Reply:
> the default of the justified-layout package that we use is 320px

Good enough for me :)

I'm pretty sure all those numbers were chosen arbitrarily. And that makes sense re 32/360/whatever. We have to draw the line somewhere.